template/src/ontology/Makefile.jinja2

```diff
@@ -249,7 +249,7 @@ OTHER_SRC = {% if project.use_dosdps -%} $(PATTERNDIR)/definitions.owl{% endif -
 
 
 $(ONTOLOGYTERMS): $(SRC) $(OTHER_SRC)
-	$(ROBOT) query --use-graphs true -f csv -i $< --query ../sparql/{{ project.id }}_terms.sparql $@
+	$(ROBOT) query --use-graphs true -f csv -i $< --query ../sparql/{{ project.id }}_terms.ru $@
```

Comment:
I don't think this line was meant to be changed to `.ru`

Reply:
Lol ooooops. Thanks